### ISSUE TITLE
Latest version of Chrome breaks first password entry

### ISSUE THREAD

**Issue by davidje13** [pivotal/usb-login-scripts#11]
The secure method used to enter the password does not work in the latest version of Chrome. Looks like it blocks typing into the developer console, or perhaps has changed the focus ordering.

**Comment by cjcjameson** [comment]
Based on things like this, I personally am intending to switch over to `scripts-autoexpire`. I don't even need to have the new chrome profile often, as I typically rotate among a small set of workstations.

I spoke with a few other people, and we're wondering if `scripts-full` is long-term maintainable. I suggest maybe removing it for now so that the `scripts-autoexpire` is more highlighted as super-solid and a worthwhile improvement over Tammer's original blog post

**Comment by davidje13** [comment]
@cjcjameson emphasising stable features by default seems reasonable, but it would be nice to keep the other stable bits (automatically adding initials to the git author file, self-update) as well as the improved commandline flags.

How about if Chrome login becomes a non-default feature in `full`? Right now you can disable it with `-L` (skip login), but it would be easy to swap that around to be off by default, and require `-kdle` (keys, duet, login, eject) to forcibly enable it.

Personally I still use `full` along with browser login, because despite this annoyance it still saves me time. I agree that it's probably going to have long-term maintenance issues (Google creating a for-sale, for-business version of Chrome certainly doesn't bode well for them offering better profile sharing capability in the basic version), and they've stated that automation integration is deprecated :( but if I get ample amounts of beach time again I'll be putting some effort into fixing up the flow as best I can (it could always be solved by simply typing the password via a virtual keyboard, like it does with the username, but that's super-risky!)

**Comment by cjcjameson** [comment]
Oh, my bad! I thought that all the other stable bits you mention (git-author, self-update, etc.) were all baked in `scripts-autoexpire`. Hm. Maybe OK for now -- I'm like you at the moment, I use full and always skip through the Chrome part.

**Comment by davidje13** [comment]
I've updated the scripts (and readme) to have Chrome login default off instead of default on. Seems like a good model to follow for any future features that might be added; off until guaranteed stable.

**Comment by davidje13** [comment]
This has been fixed in Chrome 61. Not sure if it's a feature they're supporting though, so it may break again in the future.

(also it doesn't automatically press the sign-in button now, but that's probably an easy fix)
